tests/virt_autotest/login_console.pm

```diff
@@ -36,6 +36,25 @@ sub set_ssh_console_timeout_before_use {
     send_key 'ret';
 }
 
+sub config_ssh_client {
```

Comment:
This func works better with x86)64 and aarch64 due to ipmi backend. we'd better and s390x test run as the validation run to confirm this func.

Reply:
It seems that you need add some support on s390 tests. `script_run()` time out on https://openqa.suse.de/tests/14423464#step/install_package/3